assets/js/admin-script.js

```diff
@@ -299,6 +299,23 @@ jQuery( function( $ ) {
 		triggerPreview();
 	} );
 
+	function settingsChanged( event, duration ) {
```

Comment:
I think we should add a custom trigger for `settingsChanged` too, as a replacement/companion for this one:
https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/586c9eeb6bcb5a959be0ba48c9f762fd7b315638/assets/js/admin-script.js#L287-L290

This is currently used to allow extension settings (such as the Premium Templates customizer) to trigger an update and it should allow to reveal your button too.

Reply:
Please open an issue in the Premium Templates repo after implementing this!

Reply:
https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/5354e4d863b78f56db26fdb7bea2558a0a12e7a5/assets/js/admin-script.js#L286-L289